.github/composites/install/action.yaml

```diff
@@ -27,7 +29,7 @@ runs:
       id: cache
```

Comment:
`uv` has built in support for caching, is there a reason we aren't using that?

Reply:
our existing caching setup is more advanced than what uv does.

my understanding is that uv caches dependency downloads, in this case we're caching the entire virtualenv, and post restore assure validity. this saves us from having to extract thousands of files from eggs.